docs/conf.py

```diff
@@ -231,9 +231,10 @@ def generate_r_docs(app):
         r-jsonlite=1.5=r351h96ca727_0 \
         r-matrix=1.2_14=r351h96ca727_0 \
         r-testthat=2.0.0=r351h29659fb_0 \
-        cmake=3.14.0=h52cb24c_0
+        cmake=3.14.0=h52cb24c_0 \
+        ca-certificates=2019.11.27=0
     /home/docs/.conda/bin/conda install -q -y -n r_env -c conda-forge \
-        r-pkgdown=1.3.0=r351h6115d3f_1000 \
+        r-pkgdown=1.3.0=r35h6115d3f_1001 \
```

Comment:
@StrikerRUS what if we just removed build numbers and did  `r-pkgdown=1.3.0`? Won't we have to fix this again if the RTD build machines change?

Reply:
It may fail. In some cases even builds are important and with one number everything is OK, while with another one build crashes.

Sorry, I didn't get it: how RTD build machines are related to conda packages build numbers?

Reply:
That string has some extra details in it that some packages use for builds that used, say, specific compilers for creation.

From [conda docs](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html):

> Differentiates builds of packages with otherwise identical names and versions, such as:

I've found in the past that my conda env files are not portable across operating systems until I remove those.

This is why `conda  env export` ships with the argument `--no-builds`, so you can build a conda environment file on one machine and feel confident it will work on others

![image](https://user-images.githubusercontent.com/7608904/73581968-cc119c00-4450-11ea-9dba-a39a4645ce1d.png)

Reply:
Usually build increment indicates bug fixes:
https://github.com/conda-forge/r-pkgdown-feedstock#updating-r-pkgdown-feedstock
> In order to produce a uniquely identifiable distribution:
If the version of a package is not being increased, please add or increase the build/number.
If the version of a package is being increased, please remember to return the build/number back to 0.

Reply:
I see, ok guess I misunderstood. Thanks

Reply:
From the link you have provided above:
> 

build_number | integer | A non-negative integer representing the build number of the package. Unlike the build string, the build_number is inspected by conda. Conda uses it to sort packages that have otherwise identical names and versions to determine the latest one. This is important because new builds that contain bug fixes for the way a package is built may be added to a repository.
-- | -- | --

Reply:
Despite that it seems we talked about different "build numbers" (you about `build`, me - `build_number`) and I have no idea how to specify `build_number` without `build` string, I guess you were confused by the world "inspects".

> Nothing in conda actually inspects the build string.

Maybe conda doesn't inspect build strings, it compares them foe sure. Otherwise, the following example will not work:
> The build string constraint `"numpy=1.11.2=*nomkl*"` matches the NumPy 1.11.2 packages without MKL but not the normal MKL NumPy 1.11.2 packages.
https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#examples